pubsub/google/cloud/pubsub_v1/subscriber/_protocol/bidi.py

```diff
@@ -481,6 +481,11 @@ def _thread_main(self):
                 '%s caught unexpected exception %s and will exit.',
                 _BIDIRECTIONAL_CONSUMER_NAME, exc)
 
+        else:
```

Comment:
Is self._bidi_rpc.is_active being false really unexpected?

Reply:
Yep, it really shouldn't happen. This is to help produce more logs to debug.

Reply:
SGTM. I just didn't know if 'unexpectedly' was a bit strong or not.